src/DOMINOSEC8_fmt_plug.c

```diff
@@ -541,22 +541,27 @@ static void decode(unsigned char *ascii_cipher, unsigned char *binary)
 				out <<= 6;
 				ch = *ascii_cipher;
 
-				if (ch < '0' || ch > '9')
-					if (ch < 'A' || ch > 'Z')
-						if (ch < 'a' || ch > 'z')
-							if (ch != '+')
-								if (ch == '/')
+				if (ch < '0' || ch > '9') {
+					if (ch < 'A' || ch > 'Z') {
+						if (ch < 'a' || ch > 'z') {
+							if (ch != '+') {
+								if (ch == '/') {
 									out += '?';
-								else
-								{ ; } /* shit happens */
-							else
+								} else {
+									/* shit happens */
+								}
+							} else {
```

Comment:
I'm not sure the dropped `;` would go unnoticed with any compiler, doesn't some complain about empty braces? OTOH with brackets everywhere else we can drop lines 550-551 altogether because it's no longer gets ambiguous.

Reply:
I have not seen warnings about empty braces, but I can do the changes without removing the ; and putting everything in the same line: { ; } /* shit happens */. I submitted the patch because I would use the brackets in my code, but let me know if it makes sense according to the project's guidelines.

Reply:
Like I said it's better to drop that else clause altogether. It was only there because it had to as long as no braces were used.